tests/std/tests/VSO_0000000_regex_use/test.cpp

```diff
@@ -669,6 +669,43 @@ void test_gh_5160() {
     neg_regex.should_search_fail(L"xxxYxx\x2009xxxZxxx"); // U+2009 THIN SPACE
 }
 
+void test_gh_5192() {
+    // GH-5192: Correct characters not matched by special character dot
+    using namespace string_literals;

```

Comment:
This is unnecessary - the test begins with `using namespace std;` which makes all of the UDLs available.

(Due to the two layers of `inline namespace`s, `using namespace std::string_literals;`, `using namespace std::literals;`, or `using namespace std;` makes `std::literals::string_literals` available.)